.github/workflows/pull-request.yaml

```diff
@@ -71,3 +62,38 @@ jobs:
       run: |
         export EC_VERSION="dev-metadata"
         ./scripts/cache-files.sh
+
+    - name: Cache files for integration test
```

Comment:
Why is this step duplicated ?

Reply:
Because it's cashing to two different places